index.js

```diff
@@ -1,6 +1,7 @@
 'use strict';
 
 global.D2LAccessTokenExpiresAt = 0;
+global.D2LOAuth2Disabled = false;
```

Comment:
Don't know if you actually need to attach this to the `global` object (or the expiry time variable for that matter).